app/build.gradle.kts

```diff
@@ -6,12 +6,12 @@ plugins {
 
 android {
     namespace = "hr.dice.filipbionda.tmdbpractice"
-    compileSdk = 34
+    compileSdk = 35
 
     defaultConfig {
         applicationId = "hr.dice.filipbionda.tmdbpractice"
         minSdk = 29
-        targetSdk = 34
+        targetSdk = 35
```

Comment:
because I started using new emulator with api level 35. After that I had error here so I've done that. Is it something bad?

Reply:
It is ok for now because you don't really have any implementation but version updates contain changes that could be related to your code in the app and so by updating a dependency you can introduce a bug / crash in your app. Updating versions is a task for itself!